README.adoc

```diff
@@ -56,7 +50,12 @@ To use an existing Yocto build directory:
 [source,console]
 source ./setup-environment build
 
-To start a simple image build:
+If using a fresh docker image, your locale might not be set to en_US.UTF8.
+Follow the instructions https://github.com/RobertBerger/meta-mainline/issues/8[here] to set it, otherwise your build will error.
+
+The build should be started as a normal user and NOT as a superuser. The coreutils build will error otherwise.
+If you're building from a docker image, you're logged-in as a superuser by default. Create a new user and switch to it before starting the build.
+Finally, to start a simple image build:
```

Comment:
Put this info in a docker specific part. In a non-docker setup nobody will hopefully try to invoke bitbake as root user.